Document/0x06b-Basic-Security-Testing.md

```diff
@@ -401,10 +435,46 @@ PID  Name
 499  Gadget
 ~~~
 
+![Frida on non-JB device](/Document/Images/Chapters/0x06b/fridaStockiOS.png "Frida on non-JB device")
+
 ##### Troubleshooting.
 
 If something goes wrong (which it usually does), mismatches between the provisioning profile and code signing header are the most likely suspect. In that case it is helpful to read the official documentation and gaining an understanding of how the whole system works [7][8]. I also found Apple's entitlement troubleshooting page [9] to be a useful resource.
 
+
+### Setting up Burp
+Setting up burp to proxy your traffic through is pretty straightforward. It is assumed that you have both: iDevice and workstation connected to the same WiFi network where client to client traffic is permitted. If client-to-client traffic is not permitted, it should be possible to use usbmuxd [18] in order to connect to burp through USB. 
+
+The first step is to configure proxy of your burp to listen on all interfaces (alternatively only on the WiFi interface), as per screenshot.
+
+![Setting up Burp Proxy](/Document/Images/Chapters/0x06b/setBurpProxy.png "Setting up Burp Proxy")
+
+Then we can configure our iDevice to use our proxy in advanced wifi settings. 
+![Setting up Proxy on iDevice](/Document/Images/Chapters/0x06b/setProxyiDevice.png "Setting up Burp Proxy")
+
+### Bypassing Certificate Pinning
+Certificate Pinning is a practice used to tighten security of TLS connection. 
+When an application is connecting to the server using TLS, it checks if the server's certificate is signed with trusted CA's private key. The verification is based on checking the signature with public key that is within device's key store. This in turn contains public keys of all trusted root CAs.
+Certificate pinning means that our application will have server's certificate or hash of the certificate hardcoded into the source code. 
+This protects against two main attack scenarios:
+* Compromised CA issuing certificate for our domain to a third-party
+* Phishing attacks that would add a third-party root CA to device's trust store
+
+If you think that you trust all root CAs, review the  [EFF's Observatory](https://www.eff.org/pl/observatory) and  [map of the 650-odd organizations that function as Certificate Authorities trusted (directly or indirectly) by Mozilla or Microsoft](https://www.eff.org/files/colour_map_of_CAs.pdf).
```

Comment:
What do you want to express with this last sentence and the reference to EFF? The information there seems quite outdated. Is this still maintained?

If it is maintained, you could move this last sentence to the recommendation section for the SSL Pinning test case.  Otherwise we should delete it.